assets/styles/cloud.css

```diff
@@ -3,30 +3,65 @@
 @tailwind utilities;
 
 [v-cloak] {
-    display: none;
+  display: none;
 }
 
 .shake {
-    animation: shake 0.82s cubic-bezier(.36,.07,.19,.97) both;
-    transform: translate3d(0, 0, 0);
-    backface-visibility: hidden;
-    perspective: 1000px;
+  animation: shake 0.82s cubic-bezier(0.36, 0.07, 0.19, 0.97) both;
+  transform: translate3d(0, 0, 0);
+  backface-visibility: hidden;
+  perspective: 1000px;
 }
 
 @keyframes shake {
-    10%, 90% {
-        transform: translate3d(-1px, 0, 0);
-    }
+  10%,
+  90% {
+    transform: translate3d(-1px, 0, 0);
+  }
 
-    20%, 80% {
-        transform: translate3d(2px, 0, 0);
-    }
+  20%,
+  80% {
+    transform: translate3d(2px, 0, 0);
+  }
 
-    30%, 50%, 70% {
-        transform: translate3d(-4px, 0, 0);
-    }
+  30%,
+  50%,
+  70% {
+    transform: translate3d(-4px, 0, 0);
+  }
 
-    40%, 60% {
-        transform: translate3d(4px, 0, 0);
-    }
+  40%,
+  60% {
+    transform: translate3d(4px, 0, 0);
+  }
+}
+
+/* Custom Scrollbar */
+#workshop-exercises {
+  --sb-track-color: none;
+  --sb-thumb-color: #be185d;
+  --sb-size: 5px;
+
+  scrollbar-color: var(--sb-thumb-color) var(--sb-track-color);
+}
+
+#workshop-exercises::-webkit-scrollbar {
+  width: var(--sb-size);
+}
+
+#workshop-exercises::-webkit-scrollbar-track {
+  background: none;
+  border-radius: 100vw;
+}
+
+::-webkit-scrollbar-thumb {
+  background: var(--sb-thumb-color);
+  border-radius: 100vw;
+}
+
+@supports (scrollbar-color: pink grey) {
+  #workshop-exercises {
+    scrollbar-color: #be185d #1f2937;
```

Comment:
Is this a tailwindcss colour or one of our custom colours? I guess if its a custom one we should probably define as a variable with a name, and if tailwind, use the correct class instead

Reply:
It is the same colour pink you use  just as it's hex value. So it is the same as pink-600 or something similar I can't remember without looking it up. 

I was trying to see if I could do the scroll bar how I know to do it. I will use tailwind classes next to keep in line with the project